coordinator/src/bin/coordinator.rs

```diff
@@ -261,7 +266,28 @@ async fn main() -> Result<()> {
 
     let notification_service = NotificationService::new(opts.fcm_api_key);
 
-    let _sender = notification_service.get_sender();
+    tokio::spawn({
+        let sender = notification_service.get_sender();
+        let pool = pool.clone();
+        async move {
+            loop {
+                match pool.get() {
```

Comment:
Keeping the connection up and utilising it once per hour IMHO is more wasteful. 
It should be more optimal to reach for it when actually using it, if it's done so infrequently, so that connection can be used somewhere else in the meantime.

that's my understanding at least, please correct me if I'm wrong!

Reply:
I think you are right. Keeping a connection idle for a longer period of time is less efficient than requesting a new connection every time we need it. 👍